lib/src/main/java/com/diffplug/spotless/markdown/FlexmarkStep.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2016-2021 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.spotless.markdown;
+
+import java.io.Serializable;
+import java.lang.reflect.Constructor;
+import java.util.Objects;
+
+import com.diffplug.spotless.FormatterFunc;
+import com.diffplug.spotless.FormatterStep;
+import com.diffplug.spotless.JarState;
+import com.diffplug.spotless.Provisioner;
+
+/** A step for <a href="https://github.com/vsch/flexmark-java">flexmark-java</a>. */
+public class FlexmarkStep {
+	// prevent direct instantiation
+	private FlexmarkStep() {}
+
+	private static final String DEFAULT_VERSION = "0.62.2";
+	private static final String NAME = "flexmark-java";
+	private static final String MAVEN_COORDINATE = "com.vladsch.flexmark:flexmark-all:";
+
+	/** Creates a formatter step for the default version. */
+	public static FormatterStep create(Provisioner provisioner) {
+		return create(defaultVersion(), provisioner);
+	}
+
+	/** Creates a formatter step for the given version. */
+	public static FormatterStep create(String version, Provisioner provisioner) {
+		Objects.requireNonNull(version, "version");
+		Objects.requireNonNull(provisioner, "provisioner");
+		return FormatterStep.createLazy(NAME,
+				() -> new State(JarState.from(MAVEN_COORDINATE + version, provisioner)),
+				State::createFormat);
+	}
+
+	public static String defaultVersion() {
+		return DEFAULT_VERSION;
+	}
+
+	private static class State implements Serializable {
+		private static final long serialVersionUID = 1L;
+
+		/** The jar that contains the eclipse formatter. */
```

Comment:
Stray comment.

Reply:
This was copied from FreshmarkStep and it also exists on `KtLintStep`, `KtFmtStep` and `GoogleJavaFormatStep`.  I imagined the Jar always receives some eclipse formatter as well and this comment helped remind of that. If that's not the case, I can remove it. Should I remove it from the other classes as well?

Reply:
> Should I remove it from the other classes as well?

If you don't mind, that would be great! It does not contain any eclipse stuff for `KtLint`, `KtFmt`, nor `GJF`.